test/codegen-generated-classes-test/pom.xml

```diff
@@ -274,6 +274,12 @@
             <version>${awsjavasdk.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>http-auth-aws-crt</artifactId>
```

Comment:
Added CRT dependency to be able to run sigV4a tests (otherwise it'd throw CRT not available error). There is a module to test functionalities w/o CRT https://github.com/aws/aws-sdk-java-v2/tree/master/test/crt-unavailable-tests